routes/habitRoutes.js

```diff
@@ -33,4 +35,21 @@ router.post('/create', async (req, res) => {
 		.catch(err => res.status(401).json({ err }));
 });
 
+/**
+ * @route   DELETE api/habit/:id
+ * @desc    Deletes habit
+ * @access  Private
+ */
+router.delete('/:id', (req, res) => {
+	// TODO: Authentication
+	Habit.findById(req.params.id)
+		.then(habit => {
+			// TODO: Check whether habit belongs to current authenticated user
+
+			// Delete the habit
+			habit.remove().then(() => res.json({ success: true }));
```

Comment:
It would be useful if we receive back deleted habit id, alongside success confirmation, we could use this on the frontend as confirmation from the backend what habit is exactly deleted instead of relying on passed id argument.

Reply:
Good idea, we can send back the whole habit since it will no longer be accessible in the database.